vector-im/element-android
# [Device management] Verify another session (PSG-722)

## Type of change


## Content


Handling click on the "Verify" button in the session overview screen for other session than the current one.

## Motivation and context


Closes #7143 

## Screenshots / GIFs





No UI changed.

## Tests



- Enable the new device management feature flag
- Go to Settings -> Security & Privacy -> Show All Sessions (V2, WIP)
- Find another session which is not verified
- Go to the session overview screen
- Check the "Verify" button is only visible when the current signed in session is verified
- Press the "Verify" button
- Check the verification request is received in the other session

## Tested devices

- OS version(s): Android 11

## Checklist
